initrd/init

```diff
@@ -89,6 +96,13 @@ if [ "$boot_option" = "r" ]; then
 	exit
 fi
 
+# Override CONFIG_TPM and persist via user config
+if [ -e /dev/tpm0 ]; then
+  echo "CONFIG_TPM=y" >> /etc/config.user
+else
+  echo "CONFIG_TPM=n" >> /etc/config.user
+fi
+
 combine_configs
```

Comment:
Combined in ram, no change detected.

Reply:
Point here is what is in cbfs is measured, but not the result of combine_configs.